Makefile

```diff
@@ -0,0 +1,60 @@
+.PHONY: test
+
+VERSION:=$(shell npm view '@holochain/tryorama' version)
+
+get-version:
+	@echo 'Version: '${VERSION}
+
+install-hc:
+	./ci_scripts/install-holochain.sh
+
+test:
+	make install-hc
+	make test-all
+
+test-all:
+	./ci_scripts/run-test.sh
+
+#############################
+# █░█ █▀█ █▀▄ ▄▀█ ▀█▀ █▀▀ ▄▄ █▀ █▀▀ █▀█ █ █▀█ ▀█▀ █▀
+# █▄█ █▀▀ █▄▀ █▀█ ░█░ ██▄ ░░ ▄█ █▄▄ █▀▄ █ █▀▀ ░█░ ▄█
+#############################
+# How to update holochain?
+# make HC_REV="HC_REV" update-hc
+# Example use: make HC_REV="f0e38fd9895054115d8755572e29a5d3639f69e6" update-hc
+# Note: After running this we should run the tests and check
+
+update-hc:
+	make HC_REV=$(HC_REV) update-hc-sha
+	git add nixpkgs.nix
+	git commit -m hc-rev:$(HC_REV)
+	git push origin HEAD
+
+update-hc-sha:
+	@if [ $(HC_REV) ]; then\
+		echo "⚙️  Updating tryorama using holochain rev: $(HC_REV)";\
+		echo "✔  Replacing rev...";\
+		sed -i '3s/.*/REV=$(HC_REV)/' ./ci_scripts/install-holochain.sh;\
+	else \
```

Comment:
Need to add replacing of rev in `README.md` and `test/e2e/fixture/zomes/test/Cargo.toml`